helpers/browser/driver.js

```diff
@@ -144,7 +144,24 @@ class ChromeProtocol {
     return new Promise((resolve, reject) => {
       Promise.resolve()
       .then(_ => this.sendCommand('Page.enable'))
-      .then(_ => this.sendCommand('Page.navigate', {url: url}))
+      .then(_ => this.sendCommand('Page.getNavigationHistory'))
+      .then(navHistory => {
+        const currentURL = navHistory.entries[navHistory.currentIndex].url;
+
+        // Because you can give https://example.com and the browser will
+        // silently redirect to https://example.com/ we need to check the match
+        // with a trailing slash on it.
+        //
+        // If the URL matches then we need to issue a reload not navigate
+        // @see https://github.com/GoogleChrome/lighthouse/issues/183
+        return (currentURL === url || currentURL === url + '/');
```

Comment:
since this is just returning a boolean (not resolving with it), I'd rather these two `then`s be combined.